### PR TITLE
Modal Component Added.

### PR DESCRIPTION
**Pull request by sameep-s**
What's Added:
Simple modal style has been added.
click [here](https://deploy-preview-9--aphro-css.netlify.app/components/modal) for preview.

p.s. Ignore the button styling, buttons will get styled as soon as i merge button branch. kindly ignore if already styled.

### CODE REVIEW

**Review by saicharan-gouru — APPROVED** [review]
Kindly consider the mentioned suggestions.
-UI looks cool.
-Code structure can be beautified(like removing gaps etc..)
-Please add screenshot or gif for reference.
-You can also add deploy preview link.

**Review by arjunbharti — COMMENTED** [review]
Looks good, go ahead!

File: components/modal.html [hunk 1]
```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Documentation Page- Navigation</title>
+    <link rel="stylesheet" href="../css/comp.css" />
+    <link
```
Comment (saicharan-gouru) [inline components/modal.html:9]: You can also add "type" attribute here.

File: components/modal.html [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+      href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/default.min.css"
+    />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <!-- topbar -->
+    <div class="topbar-container">
```
Comment (saicharan-gouru) [inline components/modal.html:20]: Naming can be improved. You can name it like nav bar.

File: css/comp.css [hunk 3]
```diff
@@ -589,3 +590,47 @@
   color: var(--dark);
 }
 
+/* ---------------------------------------- Modal */
+
+.modal-wapper {
+  width: 50rem;
+  font-size: 2rem;
+  border: 0.2rem solid var(--darker);
```
Comment (saicharan-gouru) [inline css/comp.css:598]: you can use px instead of rem for smaller values.

**Pull request merged**